ml-agents/mlagents/trainers/tests/simple_test_envs.py

```diff
@@ -57,9 +57,11 @@ def __init__(
         self.rewards: Dict[str, float] = {}
         self.final_rewards: Dict[str, List[float]] = {}
         self.step_result: Dict[str, BatchedStepResult] = {}
+        self.agent_id: Dict[str.int] = {}
```

Comment:
```suggestion
        self.agent_id: Dict[str, int] = {}
```
Very surprised that mypy didn't complain about that (but it might skip tests)